learning_resources_search/serializers.py

```diff
@@ -79,6 +81,8 @@ def serialize_learning_resource_for_update(
     return {
         "resource_relations": {"name": "resource"},
         "created_on": learning_resource_obj.created_on,
+        "is_learning_material": learning_resource_obj.resource_type
+        not in [COURSE_TYPE, PROGRAM_TYPE],
         **serialized_data,
     }
```

Comment:
These three fields:
1. resource_relations
2. created_on
3. is_learning_material

are "extra". None of them show up in http://localhost:8063/api/v1/learning_resources/.

(1) and (2) do not show up in http://localhost:8063/api/v1/learning_resources_search/, but (3) does. Do you know why?

Mostly asking out of curiosity. If removing (3) is easy, that might be worthwhile since then the response formats will match exactly, which has been the goal. That said, having (3) is not a problem—it's "extra" and it's not usable on the frontend since it doesn't show up in the openapi (good).

Reply:
I hid the is_learning_material from the results. It's pretty redundant anyway. I guess it's a bit weird to filter by something that's not a field in the results but keeping the the output the same is good and cluttering the results of both with a field that's pretty specific to our ui seems unnessesary